client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
```

Comment:
We actually won't require an API call for logout. We can instead just remove the `token` cookie and then redirect the user back to the login page

Reply:
I'll be working with Karandeep in the work session to implement this.